chart/k8gb/values.yaml

```
@@ -62,3 +62,6 @@ route53:
 
 ns1:
   enabled: false
+  # optional custom NS1 API endpoint for on-prem setups
+  # endpoint: http://ns1.example.com
```

@k0da IIRC, NS1 endpoint setting should not contain URI scheme
```suggestion
  # optional custom NS1 API endpoint, defaults to managed NS1 (api.nsone.net)
  # endpoint: ns1.example.com
```

---

app.Flag("ns1-endpoint", "When using the NS1 provider, specify the URL of the API endpoint to target (default: https://api.nsone.net/v1/)").Default(defaultConfig.NS1Endpoint).StringVar(&cfg.NS1Endpoint)

---

Yep, you're right, just checked in NS1 Golang SDK https://github.com/ns1/ns1-go/blob/v2/rest/client.go#L40, should be full URL.
Anyway, still makes sense to point to the default value

---

yes, thought the same, amended

---

cool 👍